kentikapi/internal/utils/convert.go

```diff
@@ -80,3 +86,11 @@ func ConvertOrNone(input interface{}, convertFunc ConvertFunc, output interface{
 	vOutput.Elem().Set(results[0])
 	return nil
 }
+
+func ParseISO8601Timestamp(t testing.TB, timestamp string) *time.Time {
```

Comment:
I think we should separate testing code from implementation.
Please put this function to separate testutil package just like here: https://github.com/kentik/community_sdk_golang/blob/6d0df03855e249bd2a28edf790a6bab17eb63f01/kentikapi/internal/testutil/format.go

Reply:
I reconsidered using this function in my_kentik_portal api tests, and leave it untouched compared to master branch.

Reply:
Great. Resolved